resources/qml/TimelineView.qml

```diff
@@ -261,7 +282,12 @@ Page {
 
 							MouseArea {
 								anchors.fill: parent
-								onClicked: chat.model.openUserProfile(modelData.userId)
+                                onClicked: {
+									if(userProfile) userProfile.destroy()
+									var component = Qt.createComponent("UserProfile.qml");
+									userProfile = component.createObject(timelineRoot,{user_data : modelData});
```

Comment:
Can you help me understand what you're trying to do?  destroying the component and reloading it from the qml file seems unnecessarily intensive to me.

Reply:
So many a times when I switch back to timeline the userprofile goes behind and clicking on the avatar just keeps on creating new instances without destroying the previous one, so I though why not destroy the previous instance and create a new one which pops up.